lib/server.js

```diff
@@ -29,6 +29,18 @@ function handleResponse (opts, req, resp, response) {
         response.headers['Access-Control-Allow-Methods'] = 'GET';
         response.headers['Access-Control-Allow-Headers'] = 'accept, content-type';
 
+        if (!/^application\/json/.test(response.headers['content-type'])) {
+            // Set up security headers
+            // https://www.owasp.org/index.php/List_of_useful_HTTP_headers
+            response.headers['X-XSS-Protection'] = '1; mode=block';
+            response.headers['X-Content-Type-Options'] = 'nosniff';
+            response.headers['Content-Security-Policy'] =
+                "default-src 'none'; media-src *; img-src *; style-src *";
+            response.headers['X-Content-Security-Policy'] = "default-src 'none'";
```

Comment:
According to http://caniuse.com/#search=csp the X-prefixed variant is still needed for IE 10 and 11. So yeah, should have the same value.